core/src/main/java/org/opensearch/sql/expression/DSL.java

```diff
@@ -341,6 +341,15 @@ public static FunctionExpression day_of_week(
     return compile(functionProperties, BuiltinFunctionName.DAY_OF_WEEK, expressions);
   }
 
+  public static FunctionExpression extract(FunctionProperties functionProperties,
+                                           Expression... expressions) {
+    return compile(functionProperties, BuiltinFunctionName.EXTRACT, expressions);
+  }
+
+  public static FunctionExpression extract(Expression... expressions) {
```

Comment:
Why have two separate functions here?

Reply:
One requires function properties. The one that doesn't (bottom) calls the first one.

Reply:
Sorry, I should have clarified. For verification purposes, I was wondering if you have examples of a function call/query for each one.

Reply:
Gotcha. Thanks